pages/article/ArticleHandler.inc.php

```diff
@@ -65,6 +65,7 @@ function authorize($request, &$args, $roleAssignments) {
 
 	/**
 	 * @see PKPHandler::initialize()
+	 * @param $request Request
```

Comment:
According to PKPHandler::initialize(), the first parameter is an instance of PKPRequest that is wrong

Reply:
`Request` is an app-specific subclass of `pkp-lib`'s `PKPRequest`.

Reply:
My confusion here is that according to phpdocs it's `PKPRequest`: https://github.com/pkp/pkp-lib/blob/ad864e9960200ec1819fb9783a78d17093036baa/classes/handler/PKPHandler.inc.php#L358 but it appears that it's its subclass `Request`. It's not obvious for me, and if I didn't check I hadn't understood that I can use methods specific to the `Request` here.

Reply:
(the same for my IDE)

Reply:
This is a more general problem that we have with the extended classes across applications. I think how you've done it here is ok and the broader solution is to update `PKPHandler::initialize`. But let's not worry about it for this PR.